src/ventures-for-shared-prosperity/what-we-provide.njk

```diff
@@ -7,14 +7,103 @@ title: What the ideas42 Venture Studio can provide
 {% block masthead %}
   {{ super() }}
   <div class="hero standard-restricted-width">
-    <div class="g-max-3 g-right">
+   {%include "components/cohort-banner.njk"%}
+    <p class="hero__tag">We're with you</p>
+
       <h1 class="hero__title">
-        What we provide
+        What we provide our entrepreneurs
       </h1>
+      <p>Starting a business is risky, and for many individuals without a personal financial safety net, or access to a network of individuals with significant capital to fund their venture, the risk is simply too high. We view this as an unacceptable barrier to entry for entrepreneurship and believe that it is both unfair, and reduces the volume and quality of new businesses as many individuals with the talent, motivation and passion to start their own business are unable to. At the ideas42 Venture Studio we can’t guarantee your success, but we can reduce your risk of trying to build an amazing business, and support those entrepreneurs who don’t have access today. </p>
+
+      <div>
+      <p>
+      If you are selected to join the ideas42 Venture Studio as an entrepreneur, we will pay you a salary ($75k per year), and provide benefits for 18th months, while surrounding you with a dedicated team of business builders, and behavioral scientists to help bring your vision to life.
+      </p>
+      </div>
+    <div class="g-max-3 g-right">
+   
+
+
+    <div>
```

Comment:
This page has the `masthead` problem. I think from here down should be  in the `main` block. Also, I think you have an extra `div` here we can remove

Reply:
Fixed!